libpod/container.go

```diff
@@ -288,6 +288,15 @@ func (c *Container) Config() *ContainerConfig {
 		return nil
 	}
 
+	if c != nil {
```

Comment:
Why would anyone call c.Config() on a container which is nil? I think the caller should handle this.

Reply:
`c.networks` within `c.Config` leads to issues in the tests where for some reason calling c.networks and checking c.conf.spec.linux results in a nil pointer deref.